manifests/r/Rosalie/Rosalie'sMupenGUI/0.5.7/Rosalie.Rosalie'sMupenGUI.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Rosalie.Rosalie'sMupenGUI

```

```suggestion
PackageIdentifier: Rosalie.RosaliesMupenGUI
```

Just a suggestion - but having an apostrophe in the name might make it difficult to work with in PowerShell, as it would need to be escaped every time. 

- If the suggestion is taken, please update file name and path to match this change.